src/BasketBundle/Controller/BasketController.php

```diff
@@ -522,7 +524,19 @@ public function finalReviewStepAction()
 
         return $this->render('SonataBasketBundle:Basket:final_review_step.html.twig', [
             'basket' => $basket,
-            'tac_error' => 'POST' == $this->get('request')->getMethod(),
+            'tac_error' => 'POST' == $this->getRequest()->getMethod(),
         ]);
     }
+
+    /**
+     * @return Request
+     */
+    public function getRequest()
+    {
+        if ($this->container->has('request_stack')) {
```

Comment:
I think we can safely rewrite those to always return `return $this->container->get('request_stack')->getCurrentRequest();`

Reply:
Done